### DISCUSSION TITLE
student developer pack Asking for renewal after 5 days and  when applying for renewal it got rejected

### DISCUSSION THREAD

**Discussion by J-DHARUN** [community/community#60336] [GitHub Education]
My student developer pack got approved few days ago but it is already asking for renewal and I am not able to use the benefits. As a student it important for me please help me on this issue. Thanks

**Comment by samus-aran** [comment]
Hi @J-DHARUN  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
